rails_event_store/spec/active_job_scheduler_spec.rb

```diff
@@ -58,19 +60,21 @@ module RailsEventStore
 
         enqueued_jobs = ActiveJob::Base.queue_adapter.enqueued_jobs
         expect(enqueued_jobs.size).to eq(1)
-        expect(enqueued_jobs[0]).to eq({
-          job: MyAsyncHandler,
-          args: [
-            {
-              "event_id" => "83c3187f-84f6-4da7-8206-73af5aca7cc8",
-              "event_type" => "RubyEventStore::Event",
-              "data" => "--- {}\n",
-              "metadata" => "---\n:timestamp: 2019-09-30 00:00:00.000000000 Z\n",
-              "_aj_symbol_keys" => ["event_id", "data", "metadata", "event_type"]
-            }
-          ],
-          queue: "default"
-        })
+        expect(enqueued_jobs[0]).to include(
```

Comment:
the enqueued_job now includes a bunch of extra metadata, for example:
```
'job_class' => 'RailsEventStore::MyAsyncHandler',
'job_id' => random uuid,
'provider_job_id' => nil,
'queue_name' => 'default',
'priority' => nil,
'arguments' => [
  {
    'event_id'        => '83c3187f-84f6-4da7-8206-73af5aca7cc8',
    'data'            => "--- {}\n",
    'metadata'        => "---\n:timestamp: 2019-09-30 00:00:00.000000000 Z\n",
    'event_type'      => 'RubyEventStore::Event',
    '_aj_symbol_keys' => %w[event_id data metadata event_type],
  },
],
'executions' => 0,
'exception_executions' => {},
'locale' => 'en',
'timezone' => 'UTC',
'enqueued_at' => timestamp string,
```
I am having trouble tracking down exactly why that is getting added now, presumably due to one of the dependency bumps. I changed the test to use `include` instead of `eq` so that it will check if those keys + values exist in the job, but it will not check if it is an exact match (since it no longer is).